debug[39m: [Chain.js]: buildTransactionID - start
[34mdebug[39m: [Chain.js]: buildTransactionID - transaction_id 3a9a0b36fa8bde6c08ec0f1da405c5edc71be18ba7278651356aac2d55078442
[34mdebug[39m: [invoke-chaincode]: Sending instantiate proposal "{"chainId":"mychannel","chaincodeId":"trace1","chaincodeVersion":"v0","fcn":"init","args":["Sku","Sku654321","TraceInfo","this is genesis block"],"txId":"3a9a0b36fa8bde6c08ec0f1da405c5edc71be18ba7278651356aac2d55078442","nonce":{"type":"Buffer","data":[170,235,148,138,37,184,40,75,223,63,13,132,99,108,46,173,117,191,110,50,202,168,88,133]},"chaincodePath":"github.com/trace"}"
[34mdebug[39m: [Chain.js]: getPeers - list size: 4.
[34mdebug[39m: [Chain.js]: buildChannelHeader - type 3 chain_id mychannel tx_id NaN epoch % chaincode_id null lccc
[34mdebug[39m: [Chain.js]: buildCurrentTimestamp - building
[34mdebug[39m: [crypto_ecdsa_aes]: ecdsa signature:  negative=0, words=[16790238, 4699269, 53334091, 24717666, 41313280, 66847613, 65792417, 12416905, 46796537, 2798426, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], length=10, red=null, negative=0, words=[34989191, 38285822, 36826612, 52545674, 11454899, 60544875, 33600766, 21313556, 51900794, 1738238, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], length=10, red=null, recoveryParam=0
[34mdebug[39m: [Chain.js]: _signProposal - signature::{"type":"Buffer","data":[48,69,2,33,0,170,205,106,202,14,249,47,93,226,126,190,154,31,240,13,246,118,100,0,94,74,88,178,221,4,177,30,210,21,0,50,222,2,32,106,23,251,23,241,122,81,78,5,32,11,79,238,111,93,172,174,201,179,200,114,34,163,30,223,73,32,199,250,21,228,135]}
[34mdebug[39m: [Peer.js]: Peer.sendProposal - Start
[34mdebug[39m: [Peer.js]: Peer.sendProposal - Start
[34mdebug[39m: [Peer.js]: Peer.sendProposal - Start
[34mdebug[39m: [Peer.js]: Peer.sendProposal - Start
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:8051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7051". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Peer.js]: GRPC client got an error response from the peer "grpc://202.77.131.29:7056". Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:434:17
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[31merror[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
    at Object.callback (D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\fabric-client\lib\Peer.js:192:13)
    at D:\eclipse-workspace\jcloud-blockchain-gogs\node_modules\grpc\src\node\src\client.js:437:14
[34mdebug[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
[34mdebug[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
[34mdebug[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
[34mdebug[39m: [Chain.js]: Chain-sendPeersProposal - Promise is rejected: Error: Error: Error starting container: Failed to generate platform-specific docker build: Error returned from build: 2 "# github.com/trace
chaincode/input/src/github.com/trace/trace_chaincode.go:276: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:305: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:335: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:388: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:449: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:512: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:571: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:628: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:686: undefined: UpdateObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: undefined: QueryObject
chaincode/input/src/github.com/trace/trace_chaincode.go:731: too many errors
"
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Instantiate transaction proposal was bad
[31merror[39m: [invoke-chaincode]: Failed to send Proposal or receive valid ProposalResponse from peers.
[31merror[39m: [invoke-chaincode]: Failed to send Proposal or receive valid ProposalResponse from peers.
[31merror[39m: [invoke-chaincode]: Failed to send Proposal or receive valid ProposalResponse from peers.
[31merror[39m: [invoke-chaincode]: Failed to send Proposal or receive valid ProposalResponse from peers.
[31merror[39m: [invoke-chaincode]: Bad proposal responses. null
[31merror[39m: [invoke-chaincode]: Bad proposal responses. null
[31merror[39m: [invoke-chaincode]: Bad proposal responses. null
[31merror[39m: [invoke-chaincode]: Bad proposal responses. null
[31merror[39m: [invoke-chaincode]: Error: Bad proposal responses. null
    at Object.throwError (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\util.js:204:8)
    at hfc.newDefaultKeyValueStore.then.then.then (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\invoke-transaction.js:169:9)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[31merror[39m: [invoke-chaincode]: Error: Bad proposal responses. null
    at Object.throwError (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\util.js:204:8)
    at hfc.newDefaultKeyValueStore.then.then.then (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\invoke-transaction.js:169:9)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[31merror[39m: [invoke-chaincode]: Error: Bad proposal responses. null
    at Object.throwError (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\util.js:204:8)
    at hfc.newDefaultKeyValueStore.then.then.then (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\invoke-transaction.js:169:9)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[31merror[39m: [invoke-chaincode]: Error: Bad proposal responses. null
    at Object.throwError (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\util.js:204:8)
    at hfc.newDefaultKeyValueStore.then.then.then (D:\eclipse-workspace\jcloud-blockchain-gogs\app\client\invoke-transaction.js:169:9)
    at process._tickCallback (internal/process/next_tick.js:103:7)
Quit with err: Error: Bad proposal responses. null
